dist/button.js

```diff
@@ -1 +1 @@
> ."+He.CHECKOUTLOGO+" {\n        margin-bottom: 0px; !important;\n    }\n",cn="\n\n\n    ."+He.BUTTON+"."+He.TEXT_COLOR+"-black {\n        color: #2C2E2F;\n    }\n\n    ."+He.BUTTON+"."+He.TEXT_COLOR+"-white {\n        color: #fff;\n    }\n\n    ."+He.BUTTON+"."+He.COLOR+"-gold {\n        background: #ffc439;\n        color: #111;\n    }\n    ."+He.BUTTON+"."+He.COLOR+"-gold:hover {\n        filter: brightness(0.95);\n\n    }\n    ."+He.BUTTON+"."+He.COLOR+"-gold:focus {\n        outline: none;\n        box-shadow: 0px 0px 1px 3px #0c67ff inset;\n    }\n\n\n    ."+He.BUTTON+"."+He.COLOR+"-blue {\n        background: #0070BA;\n    }\n\n    ."+He.BUTTON+"."+He.COLOR+"-blue:hover {\n        filter: brightness(0.95);\n    }\n\n    ."+He.BUTTON+"."+He.COLOR+"-blue:focus {\n        outline: none;\n        box-shadow: 0px 0px 1px 3px #0c67ff inset;\n    }\n\n    ."+He.BUTTON+"."+He.COLOR+"-darkblue {\n        background: #003087;\n    }\n    ."+He.BUTTON+"."+He.COLOR+"-darkblue:hover {\n        filter: brightness(1.2);\n    }\n    ."+He.BUTTON+"."+He.COLOR+"-darkblue:focus {\n        outline: none;\n        box-shadow: 0px 0px 1px 3px #0c67ff inset;\n    }\n\n    ."+He.BUTTON+"."+He.COLOR+"-silver {\n        background: #eeeeee;\n        color: #111;\n    }\n    ."+He.BUTTON+"."+He.COLOR+"-silver:hover {\n        filter: brightness(1.9);\n        background: #00000020;\n    }\n    ."+He.BUTTON+"."+He.COLOR+"-silver:focus {\n        outline: none;\n        background: #00000020;\n        box-shadow: 0px 0px 1px 3px #0c67ff inset;\n    }\n";function un(e){return d("style",{nonce:e.nonce,innerHTML:"\n        "+fn+"\n        "+sn+"\n        "+cn+"\n        \n    @keyframes show-text {\n        0% { \n    max-width: 0%;\n    opacity: 0;\n    overflow: hidden;\n }\n        100% { \n    max-width: 100%;\n    opacity: 1;\n }\n    }\n\n    "})}function dn(e){an(e)}function ln(e){var n=e.style.color,r=void 0===n?"blue":n;if("blue"===r||"darkblue"===r||"black"===r)return d(Ge,{logoColor:"white"});if("gold"===r||"white"===r||"silver"===r)return d(Ge,{logoColor:"blue"});throw new Error("Unsupported color (PP logo): "+r)}function hn(e){var n=e.style.label,r=e.content;if(!n)return d(Ke,null,"Log in with PayPal");if(n)return d(Ke,null,n);throw new Error("Unsupported button label: "+(n||r))}function pn(e){var n,r=e.onClick,o=void 0===r?V:r,i=an(e),a=i.fundingSource,f=i.style,s=i.env,c=i.nonce,u=i.customLabel,l=f.shape,h=f.color,p=function(e,n){e.preventDefault(),e.stopPropagation(),e.target.blur(),o(e,t({fundingSource:a},n))};return d("div",{class:[He.CONTAINER,He.SHAPE+"-"+l,He.ENV+"-"+s].join(" ")},d(un,{nonce:c,style:f}),d("div",t({role:"button"},((n={})["data-button"]=!0,n["data-funding-source"]=a,n),{class:[He.BUTTON,He.SHAPE+"-"+l,He.ENV+"-"+s,He.COLOR+"-"+h,He.TEXT_COLOR+"-white"].join(" "),onClick:p,onKeyPress:function(e,n){13!==e.keyCode&&32!==e.keyCode||p(e,n)},tabindex:"0"}),u&&d("div",{class:He.BUTTON_LABEL},d(ln,{style:f}),d(hn,{style:f,content:u}))))}}]);
```

Comment:
Please back out dist/ changes. These are autogenerated when the module is published.

Reply:
ok will remove the dist button